experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -62,17 +61,11 @@ import {
 export interface ISharedTree extends ISharedObject, ISharedTreeView {}
 
 /**
- * The key for the special identifier field, which allows nodes to be given identifiers that can be used
- * to find the nodes via the identifier index
+ * The key for the special node identifier field,
+ * which allows nodes to be given identifiers that can be used to find the nodes via the node identifier index.
  * @alpha
  */
-export const identifierKey: GlobalFieldKey = brand("identifier");
-
-/**
- * The global field key symbol that corresponds to {@link identifierKey}
- * @alpha
- */
-export const identifierKeySymbol = symbolFromKey(identifierKey);
+export const nodeIdentifierKey: GlobalFieldKey = brand("__n_id__");
```

Comment:
In some central location, we need to document a policy on how we are allocating the namespace of global field keys. Currently I think "root" and this are the only ones we use ourselves, but we should establish a pattern.

Personally, I'd recommend https://en.wikipedia.org/wiki/Reverse_domain_name_notation as that provides a pattern we and our users could follow that is standard and collision resistant.

Another option would be to reserve a specific prefix for our own use (ex: "__").

We could also just use v4 UUIDs for all of them.

Regardless we should update root to foillow this convention.

Reply:
That sounds like a good idea. I'm happy to do it too, if I have time, hopefully as part of my work on identifier APIs. I don't think it belongs in this PR, so I've tracked a work item for it (4481).

Reply:
at least put a TODO here so people looking at this code know whats going on.

Reply:
Sure, done. The reason I touched this at all in this PR is because I wanted to at least rename it from `identifier` to `nodeIdentifier`, but figure while I was at it I would make it something a little more robust.

Reply:
please use "TODO:" when doing something we know will need to change. It makes the code more searchable. When we review our todos before stabalizing, this is something we really don't want to miss!

Reply:
oh, you did, nice :) I just missed it.